pip/tests/test_qsharp.py

```diff
@@ -72,12 +72,12 @@ def test_dump_machine() -> None:
     state_dump = qsharp.dump_machine()
     assert state_dump.qubit_count == 2
     assert len(state_dump) == 1
-    assert state_dump[2] == (1.0, 0.0)
+    assert state_dump[2] == complex(1.0, 0.0)
     qsharp.eval("X(q2);")
     state_dump = qsharp.dump_machine()
     assert state_dump.qubit_count == 2
     assert len(state_dump) == 1
-    assert state_dump[3] == (1.0, 0.0)
+    assert state_dump[3] == complex(1.0, 0.0)
 
```

Comment:
I would add a test for iteration through state_dump somewhere here, I don't think there's a test for it right now?

Reply:
Good call, I'll add an iteration case that also includes a check for membership to confirm both of those syntaxes work as expected.